lib/reverse_sentence.rb

```diff
@@ -1,6 +1,74 @@
-# A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# # A method to reverse the words in a sentence, in place.
+# # Time complexity: O(n), where n is the length of the string
+# # Space complexity: O(1)
+
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  # raise NotImplementedError
+
+  # string reverse
+  if my_sentence == nil
```

Comment:
If the code block from 9-24 is `string_reverse`, break it out of this fn and call it.